applications/asset_tracker_v2/prj.conf

```diff
@@ -59,9 +62,16 @@ CONFIG_LTE_PSM_REQ_RAT="00000011"
 # CONFIG_AWS_IOT_AUTO_DEVICE_SHADOW_REQUEST=y
 # CONFIG_AWS_IOT_MQTT_RX_TX_BUFFER_LEN=2048
 # CONFIG_AWS_IOT_MQTT_PAYLOAD_BUFFER_LEN=2048
-# CONFIG_AWS_IOT_APP_SUBSCRIPTION_LIST_COUNT=1
+# CONFIG_AWS_IOT_APP_SUBSCRIPTION_LIST_COUNT=2
 # CONFIG_AWS_IOT_CLIENT_ID_APP=y
 # CONFIG_AWS_FOTA=y
+# ## Set nRF Cloud as source for handling incoming AGPS data. This enables the application to use
+# ## nRF Cloud specific APIs to parse and inject incoming AGPS data that has indirectly
+# ## been fetched from nRF Cloud on the cloud side.
+# CONFIG_AGPS=y
+# CONFIG_AGPS_SRC_NRF_CLOUD=y
+# CONFIG_NRF_CLOUD_AGPS=y
+# CONFIG_NRF_CLOUD=y
```

Comment:
Let's address these commented-out configs for all of the Asset Tracker v2 in a separate clean-up PR and introduce separate configuraiton files for them instead.